components/insight/SRC/org/openmicroscopy/shoola/env/data/views/calls/ThumbnailLoader.java

```diff
@@ -385,18 +380,21 @@ private ThumbnailStorePrx getThumbnailStore(PixelsData pxd) throws DSAccessExcep
     }
 
     /**
-     * Returns whether a pyramid should be used for the given {@link PixelsData}.
-     * This usually implies that this is a "Big image" and therefore will
-     * need tiling.
+     * Returns whether a pyramid should be used for the given {@link PixelsData}
+     * . This usually implies that this is a "Big image" and therefore will need
```

Comment:
Slightly odd punctuation but maybe it renders okay?

Reply:
Indeed, was eclipse auto format which did that.